src/atoms/ArrowBullet.js

```diff
@@ -0,0 +1,31 @@
+import styled, { injectGlobal } from 'styled-components';
+
+injectGlobal`
+    @font-face {
+    font-family: 'Helveticons';
+    src: url('https://styleguide.dagbladet.no/fonts/helveticons.woff');
+  }
+`;
+
+const ArrowBullet = styled.a`
+    color: ${props => props.theme.colors.darkGrey};
+    font-size: ${props => props.theme.variables.headingMediumSize};
```

Comment:
Please specify line-height when you specify font-height. In the theme, there's a corresponding line height to all the font-sizes, in this case headingMediumLineHeight.

Reply:
> Another approach could be to insert a <FontIcon> here instead of hardcoding it - the upside to that would be that we could more easily add support for different icon fonts (a single site could choose which icon font to use).
Agree, this will be useful when, for instance font url is changing, so we'll have one place in code to apply adjustments. Will improve

> Please specify line-height when you specify font-height. In the theme, there's a corresponding line height to all the font-sizes, in this case headingMediumLineHeight.
Sure